torch/_inductor/codegen/cpp_prefix.h

```diff
@@ -69,3 +69,12 @@ void flag_to_float(const T* src, float* dst, int64_t n) {
     dst_u32[i] = *(src + i) ? 0xFFFFFFFF : 0;
   }
 }
+
+template <typename T>
+void flag_to_float(T src, float* dst, int64_t n) {
```

Comment:
This overload is needed to handle the broadcasting case where vectorized float mask construction with a scalar.